operators/endpointmetrics/controllers/observabilityendpoint/observabilityaddon_controller.go

```diff
@@ -555,7 +555,7 @@ func (r *ObservabilityAddonReconciler) SetupWithManager(mgr ctrl.Manager) error
 		Watches(
 			&corev1.ConfigMap{},
 			&handler.EnqueueRequestForObject{},
-			builder.WithPredicates(getPred(clusterMonitoringConfigName, promNamespace, true, true, false)),
+			builder.WithPredicates(getPred(clusterMonitoringConfigName, promNamespace, true, true, true)),
```

Comment:
But I thought it was a delete operation?

Reply:
its not, this made no difference to the test, i was just covering our bases whilst testing. works with or without. In theory we only care about Update